packages/apify/src/utils_request.ts

```diff
@@ -383,11 +384,10 @@ function ensureRequestIsDispatched(duplexStream, gotScrapingOptions) {
 }
 
 /**
- * @param {RequestAsBrowserOptions} options
  * @ignore
  * @internal
```

Comment:
one annotation is enough, i would maybe go with `@internal` but it does not really matter